backend/backupManager.js

```diff
@@ -109,8 +109,15 @@ function startBackup(manualActivation = false, callback) {
     const folderPath = path.resolve(dataAccess.getPathInUserData("/"));
     //archive.directory(folderPath, "profiles");
 
+    let varIgnore = ['overlay-resources/**', 'backups/**', 'clips/**', 'logs/**', 'overlay.html'];
+    const includeResources = settings.backupIncludeResources();
+
+    if (includeResources) {
+        varIgnore = ['backups/**', 'clips/**', 'logs/**', 'overlay.html'];
```

Comment:
Should we be including backups inside of the backups? Maybe backups are the one thing that should always be ignored.

Reply:
how verbrous would you like the variable name?

rather than checking if this is true depending on the the verbiage of include vs ignore 
i could invert or check if ignore is false
```js
let varFilesAndFildersToIgnore = ['backups/**', 'clips/**', 'logs/**', 'overlay.html'];
const includeResources = settings.backupIncludeResources();

 if (!includeResources) {
        varFilesAndFildersToIgnore = [' overlay-resources/**','backups/**', 'clips/**', 'logs/**', 'overlay.html'];
```

```js
let varIgnore = ['backups/**', 'clips/**', 'logs/**', 'overlay.html'];
const includeResources = settings.backupIncludeResources();

 if (includeResources == false) {
        varIgnore = [' overlay-resources/**','backups/**', 'clips/**', 'logs/**', 'overlay.html'];
```